.circleci/config.yml

```diff
@@ -48,3 +48,15 @@ jobs:
               python setup.py sdist bdist_wheel --universal
               twine upload -u $PYPI_USERNAME -p $PYPI_PASSWORD dist/*
             fi
+
+workflows:
+  release:
+    jobs:
+      - build:
+          context:
+            - aws-dev
+          filters:
+            tags:
+              only: /^release-.+$/
```

Comment:
Doesn't this mean it will build only when there is a release tag?

Reply:
Yeah, in prior PRs the default "workflow" ran, but for yours, it doesn't https://app.circleci.com/pipelines/github/datadotworld/data.world-py

Seems like the prerelease and release stuff should be separated instead of all of it happening as part of build ¯\_(ツ)_/¯

Reply:
I have addressed this in my latest commit